ui/main/src/scss/styles.scss

```diff
@@ -720,3 +720,13 @@ tag-input {
 }
 
 // END CUSTOM STYLE FOR DATE RANGE PICKER
+
+.opfab-loading-spinner {
+    float: right;
+    margin-right: 5%;
+    font-size: 30px;
+    border: solid 1px var(--opfab-bgcolor);
+    text-align: center;
+    background-color: var(--opfab-bgcolor);
+    color: #aaa;
+}
```

Comment:
I would prefer keeping loading spinner in css component files (event if duplication with monitoring) as there is other opfab-loading-spinner class in other components (filters, spinner and archives components) with differents settings.

And i think it is conflicting with loading spinner that appear on application loading : 

![image](https://github.com/user-attachments/assets/9d0e3a1d-3dba-443d-8f77-3a9317ce1c88)